### PR TITLE
Update quick starts per latest UI and dev preview note

### PR DESCRIPTION
**Pull request by sterobin**
@bhardesty , would you mind peer reviewing? I updated against the latest version of the UI and added the dev preview note.

### CODE REVIEW

File: getting-started/README.adoc [hunk 1]
```diff
@@ -25,6 +25,11 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 END GENERATED ATTRIBUTES
 ////
 
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users are discouraged from using Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
```
Comment (sterobin) [inline getting-started/README.adoc:32]: @bhardesty , @jciroli-rh , @jbyrne-redhat , Paul (don't see his tag), here's the dev preview note. I've added it to all 4 quick starts as the first thing after the title, before the abstract. If Pantheon doesn't do well with it coming before the abstract or if we don't like that, I can move it to after the abstract. It just seemed logical to have it be the very first thing users see. Let me know what you think.

Comment (jbyrne-redhat) [reply getting-started/README.adoc:32]: Looks good, @sterobin.

Comment (PaulRMellor) [reply getting-started/README.adoc:32]: @sterobin (Where's my tag!?). Looks good. :+1:

Comment (jciroli-rh) [reply getting-started/README.adoc:32]: LGTM

Comment (sterobin) [reply getting-started/README.adoc:32]: @PaulRMellor, :) Sorry, your tag isn't showing up for me. I also notice that I don't think you're even listed in the org mgmt as a member of the group, which is probably why. I'll check it out and add you if not. Thanks.

Comment (bhardesty) [reply getting-started/README.adoc:32]: LGTM

**Review by bhardesty — APPROVED** [review]

**Pull request merged**